docs/IndexMetadataChangesInVersion1.0.0.md

```diff
@@ -0,0 +1,89 @@
+# Index Metadata Changes in Version 1.0.0
```

Comment:
Can we name this file `QbeastFormat1.0.0`?

Because all the related documentation of the Metadata is on the `QbeastFormat.md` file, and may be best to follow the same naming convention.

Reply:
Approve for all the rest.

Reply:
Fixed.